### ISSUE TITLE
Really? But is fantastic!

### ISSUE THREAD

**Issue by adoello** [Quicksaver/Tab-Groups#558]
I updated my Firefox, but... **My addon favorite me say**:
### "This add-on is not compatible with your version of Firefox."
Are you Serious? You kill me for this. This add-on it's essentials for separate my life, for example, my personal life or my work. And too for separate my tasks...
Please. Think about this. Thanks.

**Comment by ximex** [comment]
There are other add-ons (e.g: Conex) working on this. But today there is no really good alternative.

**Comment by antonioluppi** [comment]
How do I find my lost tabs?

**Comment by Danita** [comment]
> How do I find my lost tabs?

The only way I could restore my old tabs was rolling back to Firefox 56. I too hope that @Quicksaver could port this add-on to webextensions tech because it's an integral part of my daily workflow (different tabs with documentation for each project, research, etc.).

**Comment by antonioluppi** [comment]
I did it the downgrade and I successfully restored my group tabs, thanks!

> I too hope that @Quicksaver could port this add-on to webextensions tech because it's an integral part of my daily workflow (different tabs with documentation for each project, research, etc.).

He said that he won't port. :(
https://web.archive.org/web/20170914192149/http://fasezero.com/

But I am glad of the effort that he has been putting in all these extensions.  Thanks Luís Miguel!

**Comment by coolynx** [comment]
This issue is related to webExtension port https://github.com/Quicksaver/Tab-Groups/issues/535.

**Comment by schmittlauch** [comment]
I don't know whether the same tab hiding mechanisms are used, but for restoring your tabs in 57 [these instructions](https://github.com/denschub/firefox-tabgroups/issues/60#issuecomment-322005792 ) look like they're worth a try.

**Comment by eridal** [comment]
What options do we have?

**Comment by lca-mse** [comment]
@ antonioluppi : where did you get an older firefox from? or how did you downgrade?

**Comment by mechalynx** [comment]
https://github.com/Quicksaver/Tab-Groups/issues/535#issuecomment-332584023

**Comment by lca-mse** [comment]
@MechaLynx  thx a lot.  Worked fine. Installed V56, created bookmark folders + bookmarks for all my tabgroups and now back to V57 with the bookmarks.....

**Comment by coolynx** [comment]
There is an extension for managing containers https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/

All tabs are still opened, visible in one big tab mess, but at least you can use some functionality of built-in Firefox container privacy features and have some virtual grouping for quick access of your tabs.

I use it together with https://addons.mozilla.org/en-US/firefox/addon/conex/

**Comment by Nerothos** [comment]
Use waterfox meanwhile.

**Comment by everworld2662** [comment]
FYI I installed Tree Style Tab (which, uhgr, really is the poor man's Tab Groups, but needs must etc.) as soon as I got the 57 upgrade and it magically had all of my tab groups without me needing to roll back to v.56 or anything to get them back, so you may want to try that first for restoring your old tabs.

**Comment by kesselborn** [comment]
just  a side note: conex can import tab-group backups okaish -- creating one container per tab group. But as firefox lacks tab hiding support, you are still missing the (for me) core feature of tab groups ... so sticking with older versions for now is probably the preferred way until the tab-hiding-api has landed in firefox.

**Comment by adoello** [comment]
Hi,
I'm testing https://testpilot.firefox.com/experiments/snooze-tabs
But ... only with "2" container with imagination, because 1 for use, and the other for "hidden" container.

**Comment by Uzere** [comment]
Does tab-group make auto-backups? Where I can find it to restore with conex?

**Comment by kesselborn** [comment]
@adoello I don't think it does autobackups that are the same format as the manual backups :/

**Comment by Micro-Shadi** [comment]
okay... i like the new firefox but i hate the fact that i have to live without tab groups plugin anymore :(
i used it heavily to organize my work.. any good replacements guys ? beside Conex

**Comment by kesselborn** [comment]
@Micro-Shadi none that can hide tabs at the moment (see the [the bug linked in the conex readme](https://github.com/kesselborn/conex#conex) ). The API for this functionality does not exist yet.

**Comment by GabrielDumbrava** [comment]
Is there a way to make this add-on compatible with the new version of Firefox, or there is simply too much work to do it. I really liked it.

**Comment by technodrome** [comment]
If you all like the functionality so much, why don't you start some crowdfunding campaign to bring it back? If all of you add $5, I am pretty sure it could convince someone code a very nice addon if 1000 people pitch in ;)

**Comment by GabrielDumbrava** [comment]
I'm for it. Start it and I'll donate.

**Comment by aderraik** [comment]
HI! To my surprise, my Firefox updated and bang! _Tab Groups_ and _Tab Mix Plus_ (another good tab add-on) are GONE!
This is a great work from @Quicksaver ! Any ideas?

**Comment by Akyshnik** [comment]
Goodnight, sweet prince... 
😢😢😢

**Comment by grahamperrin** [comment]
If anyone can (please) **close and lock this issue #558**, it will make things less difficult to track; less duplication/triplication/multiplication. Thanks. 

----

> … lost tabs … 

If you're still on 57 or greater – and if you're lucky (if you have a file with the required session data): 

- https://mozilla.logbot.info/firefox/20171118#c13876429 and please join discussion in a Mozilla area (this is an issue with Firefox, not with Tab Groups). 

----

All other readers, please be aware of: 

- [About porting to webExtension · Issue #535 · Quicksaver/Tab-Groups](https://github.com/Quicksaver/Tab-Groups/issues/535) (2017-01-29)

– https://github.com/Quicksaver/Tab-Groups/issues/535#issuecomment-341179257 in particular. 

----

**Comment by Pimm** [comment]
I wrote a function which converts a Tab Groups backup to some HTML, which should be easier to read. https://gist.github.com/Pimm/dbafbd12011afb0c09086fa4f930c2f5
